src/workers/postUpdated.ts

```diff
@@ -301,7 +301,6 @@ const updatePost = async ({
 
   data.id = databasePost.id;
   data.title = title;
-  data.visible = updateBecameVisible;
   data.visibleAt = updateBecameVisible
```

Comment:
Checking it one more time, @idoshamun how should `visibleAt` function, needs to be set each time post becomes visible (changes from false to true) or only the first time? Also if post is not visible it should be `null`?

Reply:
I'm ooo, so can't review completely but let me clarify the business logic:
So if a post doesn't have a title, it should have visible set to false and visible at null. Once it has a title, we set visible to true and visible at accordingly. Visible is a one way street. Once a post is visible it can't become invisible

Reply:
> Visible is a one way street. Once a post is visible it can't become invisible

This is the crucial info I was missing, and also missing from our worker.

Reply:
I do think now the title always persist from db if exist so technically couldnt ever be flipped afaik.
But indeed its not explicitly known.

Reply:
@rebelchris @idoshamun I refactored the logic, hopefully is more streamlined. All tests pass and I added to additional to validate cases Ido metnioned: 
- visible is once off, once post is visible it can't become invisible
- post becomes visible if it has the title
- post visibleAt is updated when post becomes visible (default null during creation)